src/tags.json

```diff
@@ -102,7 +102,7 @@
   "key": ["password", "login", "authentication", "secure"],
   "layers": ["stack"],
   "layout": ["window", "webpage"],
-  "life-bouy": ["help", "life ring", "support"],
+  "life-buoy": ["help", "life ring", "support"],
```

Comment:
This typo will probably have to live forever.

https://www.merriam-webster.com/dictionary/buoy
https://www.merriam-webster.com/dictionary/life%20buoy

> Definition of life buoy
> : a ring-shaped life preserver


https://www.merriam-webster.com/dictionary/bouy
> The word you've entered isn't in the dictionary. Click on a spelling suggestion below or try again using the search bar above.

Happy to drop this / file a bug / something else.